### PR TITLE
Add import file to submission on behalf(EXPOSUREAPP-9331)

### PR DESCRIPTION
**Pull request by mtwalli**
Review after https://github.com/corona-warn-app/cwa-app-android/pull/4054

- Add text message when QrCode is processing 
- Add import file functionality to on submission on behalf 

## Ticket
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9331

### CODE REVIEW

**Review by NataliaLemmerth — DISMISSED** [review]
UA approved

**Review by BMItr — APPROVED** [review]
:ok_hand:

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/qrcode/ui/QrCodeScannerFragment.kt [hunk 1]
```diff
@@ -78,7 +79,7 @@ class QrCodeScannerFragment : Fragment(R.layout.fragment_qrcode_scanner), AutoIn
                     .setOnDismissListener { popBackStack() }
                     .show()
 
-                InProgress -> binding.qrCodeScanSpinner.show()
+                InProgress -> binding.qrCodeProcessingView.isVisible = true
```
Comment (jurajkusnier) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/qrcode/ui/QrCodeScannerFragment.kt:82]: this line looks redundant to me as you are using
```kotlin
binding.qrCodeProcessingView.isVisible = scannerResult == InProgress
```

Comment (mtwalli) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/qrcode/ui/QrCodeScannerFragment.kt:82]: I know, but `when` is exhaustive if the state is not added, complier complaints

**Review by jurajkusnier — APPROVED** [review]

**Pull request merged**